tests/IntegrationTests/TaskManager.IntegrationTests/TestData/Helper.cs

```diff
@@ -30,16 +30,16 @@ public static TaskDispatchTestData GetTaskDispatchByName(string name)
             throw new Exception($"Task Dispatch {name} does not exist. Please check and try again!");
         }
 
-        public static ExecutionStatTestData GetExecutionStatsByName(string name)
-        {
-            var executionStatsTestData = ExecutionStatsTestData.TestData.Find(c => c.Name == name);
+        //public static ExecutionStatTestData GetExecutionStatsByName(string name)
+        //{
+        //    var executionStatsTestData = ExecutionStatsTestData.TestData.Find(c => c.Name == name);
 
-            if (executionStatsTestData != null)
-            {
-                return executionStatsTestData;
-            }
+        //    if (executionStatsTestData != null)
+        //    {
+        //        return executionStatsTestData;
+        //    }
 
-            throw new Exception($"Execution stat {name} does not exist. Please check and try again!");
-        }
+        //    throw new Exception($"Execution stat {name} does not exist. Please check and try again!");
```

Comment:
Think this should be removed, rather than commented out.

Reply:
done :)